playbooks/setup_forklift.yml

```diff
@@ -22,5 +25,6 @@
         state: 'present'
     - name: 'Clone Forklift'
       git:
-        repo: 'http://github.com/theforeman/forklift'
+        repo: "http://github.com/{{ github_org }}/forklift"
```

Comment:
I'd just use `forklift_url` so you're not tied to github. You can also change it to https://

Reply:
Where else would I get than from github?

Reply:
I imagine there could be instances where you have a local mirror. I also think `forklift_url` is a better naming than `github_org`.

Reply:
We have already something similar in the `bats` role:

https://github.com/theforeman/forklift/blob/96a04c3e180ee8794dfa7df30602ae6126c0caa0/roles/bats/tasks/main.yml#L33-L38

should that be moved out to an own role instead?

Reply:
(and strictly speaking, this does not belong into this PR anyways ;))